lib/iris/tests/__init__.py

```diff
@@ -91,6 +91,13 @@
     GRIB_AVAILABLE = True
     from iris.fileformats.grib.message import GribMessage
 
+try:
+    import iris_sample_data
+except ImportError:
+    SAMPLE_DATA_AVAILABLE = False
+else:
+    SAMPLE_DATA_AVAILABLE = True
```

Comment:
Yup, but for continuity I'm using the same pattern within `iris.tests.__init__` that's already established there.

Happy to change this if you want.